### PR TITLE
PATH PATH-DressupLeadInOut-Arc-Fix

### PR DESCRIPTION
**Pull request by IchGucksLive**
Thank you for creating a pull request to contribute to FreeCAD! To ease integration, please confirm the following:

- [x ] Branch rebased on latest master `git pull --rebase upstream master`
- [ x] Unit tests confirmed to pass by running `./bin/FreeCAD --run-test 0`
- [x ] Commit message is [well-written](https://chris.beams.io/posts/git-commit/)
- [ ] Commit message includes `issue #<id>` or `fixes #<id>` where `<id>` is the [associated MantisBT](https://freecadweb.org/wiki/tracker#GitHub_and_MantisBT) issue id if one exists

And please remember to update the Wiki with the features added or changed once this PR is merged.

HI i made the Final Changes to also Start on ARC and Cleand up the Dressup 
Linereducing by adding Definitions 
My internal shapes all work well also the Lathe Parts now working 
THANKS  
---

### CODE REVIEW

File: src/Mod/Path/PathScripts/PathDressupLeadInOut.py [hunk 1]
```diff
@@ -119,42 +119,45 @@ def normalize(self, Vector):
...
+            arccenter = FreeCAD.Vector(p0.x+queue[1].I,p0.y+queue[1].J, 0.0)
+            v = self.normalize(p1.sub(arccenter))
+        off_v = self.getOffVector(obj,v,R)
         offsetvector = FreeCAD.Vector(v.x*R, v.y*R, 0)  # IJ
         if obj.RadiusCenter == 'Radius':
             leadstart = (p0.add(off_v)).sub(offsetvector)  # Rmode
         else:
             leadstart = p0.add(off_v)  # Dmode
         if action == 'start':
+
```
Comment (luzpaz) [inline src/Mod/Path/PathScripts/PathDressupLeadInOut.py]: Is this "empty line" necessary ? (it actually has whitespace in it)

Comment (IchGucksLive) [reply src/Mod/Path/PathScripts/PathDressupLeadInOut.py]: of cause not 
as i see there are more then 12 lines in the old one 
i did correct them all here as i didn't relies them 
BUT as i am not Familiar with the git system How do i correct a PR Code
@luzpaz @sliptonic @wwmayer @realthunder

**Comment by IchGucksLive** [comment]
i updated the code and pushed a new commit 
do i or someone else delete the old commit or is it the auto correction that takes care and only uses the newer file code 
as i said i am not that familiar to use the system 
@luzpaz @wwmayer

**Comment by luzpaz** [comment]
@IchGucksLive git is pretty smart, and built for this type of thing. For example, the way git works is that you can [squash the commits all in to one commit](https://www.devroom.io/2011/07/05/git-squash-your-latests-commits-into-one/). This is typically done (AFAIK) in different cases, for example:  

- before submitting the PR
- if the user found a mistake themselves and corrects it after PR was submit (this can be problematic)
- once the PR has been reviewed and the reviewer has asked you to make changes
- once the PR has been reviewed and the reviewer squashes it for you before committing to master

_Note: If you squash all your commits in to one it will essentially delete the history or the process it took you to get to the final code you are submitting to the PR._

**Comment by IchGucksLive** [comment]
@luzpaz  is this edit now good for you

**Comment by IchGucksLive** [comment]
the gauching is as the USER needs to handle the CORNER Shape in the BASE  operation to SQUARE not Round 
AND ofcause the MISSING best startpoint Search from Libarea 
All Functions shoudt be best handeled as from system this is not the case on Base operation 
SO USER needs to handle this

**Pull request closed without merging**